build.gradle

```diff
@@ -35,6 +35,7 @@ subprojects {
         toolVersion = '6.13.0'
         ruleSetFiles = rootProject.files('config/pmd/ruleset.xml')
         ruleSets = []
+        consoleOutput = true
```

Comment:
Mi się wydaje, że to tutaj nie działa. W sensie zakomentowałam sobie testowo `@Suppress` i nic się nie pokazało. Chyba musi być, tak jak pisałam, w tym bloku:
```
tasks.withType(Pmd) {
    reports {
        xml.enabled = false
        html.enabled = true
        consoleOutput = true
    }
}
```